html/json/WSutils.js

```diff
@@ -17,9 +17,9 @@ function toggleButton(key, trueText, falseText) {
 function mediaSelect(key, format, type, humanName) {
 	var select = $("<select>").append($("<option value=''>No " + humanName + "</option>"));
 	WS.Register("ScoreBoard.Media.Format(" + format + ").Type(" + type + ").File(*).Name", function(k, v) {
-		select.children("[value='"+k.File+"']").remove();
+		select.children("[value='" + escape("/" + format + "/" + type + "/" + k.File) + "']").remove();
```

Comment:
escape() is for URLs, I doubt that's that appropriate for a CSS selector. jQuery has a escapeSelector() function, however escaping should be a last resort as it's prone to mistakes like these. The filter function with a function would be best here, given that jQuery lacks that sort of stuff that has become standard for dealing with SQL injections.

Reply:
How about you provide a better solution then instead of just unconstructively complaining how our fixes of a bug you introduced are insufficient?